browser-test/src/application_csv_download.test.ts

```diff
@@ -1,11 +1,15 @@
 import { startSession, logout, loginAsGuest, loginAsAdmin, ApplicantQuestions, AdminQuestions, AdminPrograms, endSession } from './support'
 
 describe('normal application flow', () => {
-  // If this times out, a likely cause is the .click() calls in
-  // support/admin_programs, which are called out as being asserts.
-  jest.setTimeout(25000);
+  // Our browser tests could be long-running (as of 2021-04-06, the longest
+  // takes 27 seconds). However, if your seletor fails to locate the HTML
+  // element, the test hangs as well. If you find the tests take unnaturally
+  // long, you want to verify that your selectors are working as expected.
+  // Because all tests are run concurrently, it could be that your selector
+  // selects a different entity from another test.

```

Comment:
FYI: This problem is well-addressed by setDefaultTimeout, which makes a hanging `.click` operation time out after 1 second.  You can probably set this timeout much lower.

Reply:
Good point, will remove the comment. I think the timeout includes the time waiting for the page to load, so let's leave it as 1 second for now.

Reply:
Move the comment to above setDefaultTimeout.